### PR TITLE
feat: emergency group ugov proto

### PR DESCRIPTION
**Pull request by robert-zaremba**
* proto messages
* server interface

### CODE REVIEW

File: proto/umee/ugov/v1/tx.proto [hunk 1]
```diff
@@ -14,9 +14,12 @@ option (gogoproto.goproto_getters_all) = false;
 service Msg {
   // GovUpdateMinGasPrice sets protocol controlled tx min fees.
   rpc GovUpdateMinGasPrice(MsgGovUpdateMinGasPrice) returns (MsgGovUpdateMinGasPriceResponse);
+
+  // GovSetEmergencyGroup sets protocol controlled tx min fees.
```
Comment (toteki) [inline proto/umee/ugov/v1/tx.proto:18]:
```suggestion
  // GovSetEmergencyGroup sets emergency group address.
```

File: tests/accs/accounts.go [hunk 2]
```diff
@@ -21,3 +22,8 @@ var (
 var (
 	FooModule = authtypes.NewModuleAddress("foomodule")
 )
+
+// MkAddress creats an address from the give seed string
```
Comment (toteki) [inline tests/accs/accounts.go:26]:
```suggestion
// MkAddress creats an address from the given seed string
```

File: tests/accs/accounts.go [hunk 3]
```diff
@@ -21,3 +22,8 @@ var (
...
+func MkAddress(seed string) sdk.AccAddress {
```
Comment (toteki) [inline tests/accs/accounts.go:27]:
```suggestion
func AddressFromSeed(seed string) sdk.AccAddress {
```

**Review by toteki — APPROVED** [review]
Check proto naming and comments 

Pre-approving

**Review by gsk967 — APPROVED** [review]
LGTM

**Pull request merged**